src/translations/en.json

```diff
@@ -1979,6 +1979,9 @@
               "learn_more": "Learn more about conditions",
               "add": "Add condition",
               "test": "Test",
+              "testing": "Testing condition...",
+              "testing_error": "Condition is not passing",
```

Comment:
Should this be present or past tense?
 - Condition is not passing
 - Condition did not pass

Reply:
You see it when it happens, it's telling something about what you are looking at right now. So I think present makes sense (and is more active?)

Reply:
Maybe this is better? Not a big fan of the -ing

Condition passes / Condition did not pass.

Reply:
Done